### PR TITLE
[KAN-17]:Feature/contact page

### PR DESCRIPTION
**Pull request by FevenSeyfu**
In this pull request, I have finished the Contact Us page, ensuring it matches the design across mobile, tablet, and desktop.

### CODE REVIEW

File: src/components/pages/ContactUs/ContactUs.jsx
Comment (keziabaidoo) [inline src/components/pages/ContactUs/ContactUs.jsx:1]: Well structured. Layout looks clean

File: src/components/pages/ContactUs/ContactUs.module.css
Comment (keziabaidoo) [inline src/components/pages/ContactUs/ContactUs.module.css:1]: The mobile view looks good from my side

**Review by keziabaidoo — COMMENTED** [review]
The mobile view looks good from my side . The code looks well-formatted and clean.

File: src/components/pages/ContactUs/ContactUs.module.css
Comment (FevenSeyfu) [reply src/components/pages/ContactUs/ContactUs.module.css:1]: Thank you for the feedback @keziabaidoo, However, I see you have not requested change nor approved the changes and please also note I have made changes to resolve conflict and to match the updated button's format.

**Comment by poulami-saha** [comment]
image
The mobile design needs a bit of refactoring

File: src/components/pages/ContactUs/ContactUs.module.css
Comment (keziabaidoo) [reply src/components/pages/ContactUs/ContactUs.module.css:1]: Thank @FevenSeyfu I will take note of that.

**Comment by FevenSeyfu** [comment]
Hi @poulami-saha  thanks, as per the comment made changes to the mobile view to maintain the contact form and topics list to be equally aligned. let me know if there is anything else I missed.
image

**Review by keziabaidoo — COMMENTED** [review]
Looks good to approve

**Review by poulami-saha — APPROVED** [review]

**Pull request merged**